### PR TITLE
Add VLAN BPF Bypass Annotation

### PR DESCRIPTION
**Pull request by mateoflorido**
## Overview
Add a new annotation to configure VLAN BPF Bypass for interfaces on the node.
## Changes
- Added a new annotation.
- Adjusted godoc format.

### CODE REVIEW

**Review by addyess — APPROVED** [review]
Alrighty!

File: api/v1/annotations/cilium/annotations.go [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
 	AnnotationDirectRoutingDevice = "k8sd/v1alpha1/cilium/direct-routing-device"
+
+	// Comma separated list of VLAN tags to bypass eBPF filtering on native
+	// devices. Cilium enables firewalling on native devices and filters
+	// all unknown traffic, including VLAN 802.1q packets, which pass
+	// through the main device with the associated tag (e.g., VLAN device
+	// eth0.4000 and its main interface eth0). Supports '0' as wildcard for
+	// bypassing all VLANs.
+	// e.g., k8sd/v1alpha1/cilium/vlan-bpf-bypass="4001,4002"
+	AnnotationVLANBPFBypass = "k8sd/v1alpha1/cilium/vlan-bpf-bypass"
```
Comment (bschimke95) [inline api/v1/annotations/cilium/annotations.go:23]: should this be EBPF or BPF? IIUC correctly eBPF is somewhat significantly different to BPF.

**Review by bschimke95 — APPROVED** [review]
LGTM, minor nit - feel free to ignore.

File: api/v1/annotations/cilium/annotations.go (hunk 1 above)
Comment (mateoflorido) [reply api/v1/annotations/cilium/annotations.go:23]: I'm using the same acronym as in the Cilium chart. However, these terms are often used interchangeably: https://ebpf.io/what-is-ebpf/#what-do-ebpf-and-bpf-stand-for

**Pull request merged**